app/views/reports/regions/who_report.csv.erb

```diff
@@ -0,0 +1,103 @@
+<%
+  csv = CSV.generate(headers: true) do |csv|
+  csv << ["Facility Report #{@period.to_date.strftime("%B %Y")}"]
```

Comment:
@kpethtel is it possible to add a new row, `section_labels` between the `title` and `month_labels` rows? Found a potential solution ([see in Stackoverflow](https://stackoverflow.com/questions/6156600/in-ruby-csv-how-to-write-a-blank-instead-of-to-a-file)):

```
section_labels = [
  nil,
  nil,
  nil,
  nil,
  nil,
  nil,
  nil,
  nil,
  nil,
  nil,
  nil,
  "New Registrations",
  nil,
  nil,
  nil,
  nil,
  nil,
  "Follow Up",
  nil,
  nil,
  nil,
  nil,
  nil,
  "Cumulative treatment outcome among patient under care",
  "Drug availability",
  nil,
  nil,
  nil
]

Reply:
Yes, definitely. I don't think it will format correctly, but I'll definitely add a row there.